src/main/java/com/wcc/platform/domain/cms/pages/events/EventsPage.java

```diff
@@ -6,11 +6,13 @@
 import com.wcc.platform.domain.cms.pages.PageData;
 import com.wcc.platform.domain.cms.pages.PageMetadata;
 import com.wcc.platform.domain.platform.Event;
+import jakarta.validation.constraints.NotNull;
 
 /** Events page details. */
 public record EventsPage(
-    PageMetadata metadata,
-    HeroSection heroSection,
-    Page page,
-    Contact contact,
-    PageData<Event> data) {}
+    @NotNull String id,
+    @NotNull PageMetadata metadata,
+    @NotNull HeroSection hero,
```

Comment:
the name should be heroSection, could you keep consistent also to make easier for the frontend.

Reply:
done